pkg/ccl/importccl/read_import_base.go

```diff
@@ -118,13 +118,21 @@ type readFileFunc func(context.Context, *fileReader, int32, string, chan string)
 func readInputFiles(
 	ctx context.Context,
 	dataFiles map[int32]string,
+	resumePos map[int32]int64,
 	format roachpb.IOFileFormat,
 	fileFunc readFileFunc,
 	makeExternalStorage cloud.ExternalStorageFactory,
 ) error {
 	done := ctx.Done()
 
 	fileSizes := make(map[int32]int64, len(dataFiles))
+	getResumePos := func(idx int32) int64 {
```

Comment:
read from nil map returns zero value so you don't need this function at all I think -- you can just do `resumePos[idx]` everywhere below.

Reply:
Done.